README.md

```diff
@@ -47,25 +45,24 @@ out our [document](http://uniqush.org/documentation/index.html).
 - Q: Then that's cool. But I noticed that you are using [Go](http://golang.org) programming language. Do I need to install [Go](http://golang.org) compiler and other stuff to run *uniqush-push*?
 - A: No. There is **no** dependency. All you need to do is to download the
   binary file from the [download page](http://uniqush.org/downloads.html) and
-install it. You only need to have a [redis](http://redis.io) database running
+install it. You only need to have a [Redis](http://redis.io) database running
 somewhere so that *uniqush-push* can store the user data in
-[redis](http://redis.io). For more details, here is the [installation guide](http://uniqush.org/documentation/install.html)
+[Redis](http://redis.io). For more details, here is the [installation guide](http://uniqush.org/documentation/install.html)
 
-- Q: This is nice. I want to give it a try. But you are keep talking about *uniqush-push*, and I'm talking about *uniqush*, are they the same thing?
+- Q: This is nice. I want to give it a try. But you are keep talking about *uniqush-push*, and I'm talking about *Uniqush*, are they the same thing?
 - A: Thank you for your support! *Uniqush* is intended to be the name of a
   system which provides a full stack solution for communication between mobile
 devices and the app's server. *uniqush-push* is one piece of the system.
 However, right now, *uniqush-push* is the only piece and others are under
-active devlopment. If you want to know more details about the *uniqush*
+active development. If you want to know more details about the *Uniqush*
 system's plan, you can read the [blog
 post](http://blog.uniqush.org/uniqush-after-go1.html). If you want to know the
-latest progress about *uniqush* system, please check [our
-blog](http://blog.uniqush.org/). And if you are really impatient, [our github
+latest progress about *Uniqush* system, please check [our
```

Comment:
Do either of you know how to update this, on an unrelated note? The last post was 1.5.2